.test-infra/jenkins/job_LoadTests_coGBK_Python.groovy

```diff
@@ -147,25 +147,30 @@ def loadTestConfigurations = { datasetName ->
         autoscaling_algorithm: 'NONE'
       ]
     ],
-  ].each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+  ]
+  .each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+  .each { test -> (mode) != 'streaming' ?: addStreamingOptions(test) }
 }
 
-def batchLoadTestJob = { scope, triggeringContext ->
-  scope.description('Runs Python CoGBK load tests on Dataflow runner in batch mode')
-  commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 240)
+def addStreamingOptions(test) {
+  // Use highmem workers to prevent out of memory issues.
+  test.pipelineOptions << [streaming: null,
+    worker_machine_type: 'n1-highmem-4'
```

Comment:
Why this type of machine?

Reply:
My pipelines kept crushing because of OutOfMemory exceptions. So I followed the advice given in this article: https://cloud.google.com/community/tutorials/dataflow-debug-oom-conditions

Other solution was to, according to the article, use fewer number of threads per worker. But I couldn't have found a pipeline option responsible for that in Python SDK (it exists in Java SDK, though).

Reply:
I see. Maybe it's worth to create an issue in jira with this bug? I am not sure it should be infra issue or maybe python if we need this option. How is it called in Java SDK?

Reply:
I wonder only what would happen if this worker won't be available when cron job starts? Or it will be permanently shut down.

Reply:
> How is it called in Java SDK?

It's `--numberOfWorkerHarnessThreads`. Some time ago, its equivalent in Python SDK was `--experimental worker_threads=[n]`, but the option was removed. I'll open a discussion on dev@ to find out if it is something the community should take care of.

Reply:
> I wonder only what would happen if this worker won't be available when cron job starts?

Workers are still managed by Dataflow. That option (`--worker_machine_type`) is just a hint about what machines should Dataflow use

Reply:
Thanks a lot Kamil :)